data_tools/schema.py

```diff
@@ -0,0 +1,97 @@
+import validators
+
+class Schema(object):
+    scheme_id = None
+    name = None
+    version = None
+    codes = []
+    documentation = {}
```

Comment:
I always get confused over whether `{}` is a dict or a set

Reply:
done